blips/blip-3-peerreview.md

```diff
@@ -27,27 +29,37 @@ This standard should specify which fields to register in every record, the forma
 |     id | string  | IPFS Hash??   | Yes     | The id should be unique. How to make sure reviews are unique? See [Zooko's triangle](https://en.wikipedia.org/wiki/Zooko%27s_triangle) for a relevant discussion on the compromises between decentralization, human usability and unique identifiers   |
 |     timestamp | uint32     | unix timestamp     |  Yes    |
 |     author | address     |      |  Yes    |
-|     journalId | string     |      |  Yes    |
-|     publisher | string     |      |  Yes    |
-|     manuscriptId | string     |     |  Yes    |
-|     manuscriptHash | string     |     |  Yes    |
-|     recommendation | enum     |     |  Yes    |
+|     journalId | string     |   ISSN    |  Yes    |
+|     publisher | string     |     Name?   |  Yes    |
+|     manuscriptId | string     |  DOI   |  Yes    |
+|     manuscriptHash | string     |  IPFS   |  Yes    |
+|     recommendation | enum (**TODO**: See other standards such as Orcid, Publons...)    |     |  Yes    |
```

Comment:
May be interesting to have an additional field that labels the type of peer review it is, i.e., double-blind, open, etc.